_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -0,0 +1,135 @@
+
+## Action items
+
+*None this week.*
+
+## News
+
+*No significant development news this week.*
+
+## Bitcoin Core PR Review Club
+
+_In this section, we summarize a recent Bitcoin Core PR Review Club meeting,
+highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting._
+
+FIXME:jnewbery (write or delete section)
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3612][] adds startup parameters `--large-channels` and
+  `--wumbo` (which are equivalent).  If used, the node will advertise
+  support for `option_support_large_channel` in its `init` message,
+  meaning it will accept channel opens with a value higher than the
+  previous limit of about 0.168 BTC.  If the remote peer also supports
+  this option, C-Lightning's `fundchannel` RPC will allow the user to
+  create channels over the previous limit.  See also Eclair's support
+  for this option described in [Newsletter #88][news88 eclair1323].
+
+- [C-Lightning #3600][] adds experimental support for *onion messages*
+  using *blinded paths*:
+
+    - *Onion messages* (called "LN direct messages" in
+      [Newsletter #86][news86 ln dm]) allow a node to send an encrypted
+      message across the network without using the LN payments
+      mechanism.  This can replace the mechanism of
+      messages-over-payments used by apps such as [Whatsat][].  Compared
+      to messages-over-payments, onion messages have several advantages:
+
+        1. They have a [draft specification][onion messages draft spec]
+           which, if adopted, will make it easier for multiple
+           implementations to support them.
+
+        2. They don't need the security of an onchain-enforceable payment
+           channel so onion messages can be routed even between peers
+           that don't share an established payment channel.  <!-- BOLT7
+           draft PR: "SHOULD accept onion messages from peers without an
+           established channel." -->
+
+        3. They don't require the bidirectional transmission of
+           information like HTLCs or error messages, so once a node has
+           forwarded a message, it doesn't need to keep any information
+           related to that message.  This minimizes the memory
+           requirements for nodes.  If the sending node wants to receive
+           a reply, the draft specification allows it to include a
+           blinded `reply_path` field that the receiving node can use to
+           send a reply in a new message.
+
+    - *Blinded paths* (called "lightweight rendez-vous routing" in
+      [Newsletter #85][news85 lw rv]) make it [possible][blinded path
```

Comment:
It looks like you changed the 'possible' link from a gist to the bolt proposal. I agree we should link to that proposal, but I don't think it makes much sense to link from the word 'possible'. How about: `(called "lightweight rendez-vous routing" in [Newsletter #85][news85 lw rv] and defined in a [draft proposal][blinded path proposal]) make it possible...`

Reply:
Oh, huh, I didn't even realize the link changed on me; last I looked it was a [gist](https://gist.github.com/t-bast/9972bfe9523bb18395bdedb8dc691faf).  Thanks for catching that, will fix.